keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/keymap.c

```diff
@@ -0,0 +1,220 @@
+            case HAPPYFACE:       // ʘ‿ʘ 
+                if(record->event.pressed){
+                     send_unicode_hex_string("0298 203F 0298");
+                }
+                return false;
+                break; 
+            case CMDCLEAR:
+                if (record->event.pressed) {
+                    register_code(KC_LGUI);
+                    tap(KC_A);                 
+                    unregister_code(KC_LGUI);
+                    tap(KC_DEL);                 
+                }
+                return false;
+                break;  
+            case SHRUG: // ¯\_(ツ)_/¯
+                if (record->event.pressed) {
+                    send_unicode_hex_string("00AF 005C 005F 0028 30C4 0029 005F 002F 00AF");
+                }
+                return false; 
+                break;
+            case HEARTFACE:       // ♥‿♥
+                if(record->event.pressed){
+                    send_unicode_hex_string("2665 203F 2665");
+                }
+                return false;
+                break;  
+            case DISFACE:       // ಠ_ಠ 
+                if(record->event.pressed){
+                    send_unicode_hex_string("0CA0 005F 0CA0");
+                }
+                return false;
+                break;
+            case TFLIP:         // (╯°□°)╯ ︵ ┻━┻ 
+                if(record->event.pressed){
+                    send_unicode_hex_string("0028 256F 00B0 25A1 00B0 0029 256F 0020 FE35 0020 253B 2501 253B");
+                }
+                return false;
+                break;
+            case TFLIP2:         // ┻━┻︵ \(°□°)/ ︵ ┻━┻  
+                if(record->event.pressed){
+                    send_unicode_hex_string("253B 2501 253B FE35 0020 005C 0028 00B0 25A1 00B0 0029 002F 0020 FE35 0020 253B 2501 253B");
+                }
+                return false;
+                break;
+                } 
+    }
+    return true;
+}
+
+
+/* tap dance time */
+void tdexample1(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING1);
+    reset_tap_dance (state);
+  }
+}
+void tdexample2(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING2);
+    reset_tap_dance (state);
+  }
+}
+void tdexample3(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING3);
+    reset_tap_dance (state);
+  }
+}
+void tdexample4(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING4);
+    reset_tap_dance (state);
+  }
+}
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [TD_EXAMPLE1] = ACTION_TAP_DANCE_FN(tdexample1),
+    [TD_EXAMPLE2] = ACTION_TAP_DANCE_FN(tdexample2),
+    [TD_EXAMPLE3] = ACTION_TAP_DANCE_FN(tdexample3),
+    [TD_EXAMPLE4] = ACTION_TAP_DANCE_FN(tdexample4)
+};
+
+// Runs just one time when the keyboard initializes.
+void matrix_init_user(void) {
+rgblight_setrgb (16, 0, 16);
```

Comment:
You may want to run this on the first matrix scan (use a boolean to detect).  Otherwise it will be overwritten. 

For instance:
```c
void matrix_scan_user(void) {
  static bool has_ran_yet;
  if (!has_ran_yet) {
    has_ran_yet = true;
    	rgblight_setrgb (16, 0, 16);
  }
}
```